cmd/run.go

```diff
@@ -75,21 +77,32 @@ func start(cliCtx *cli.Context) error {
 		}
 	}()
 
-	rollupID := getRollUpIDIfNeeded(components, cfg.NetworkConfig.L1Config, l1Client)
+	ethermanClient, err := etherman.NewClient(cfg.L1NetworkConfig,
```

Comment:
Just check if this can be instantiated only if needed.

Reply:
+1